server/controllers/finance/accounts/index.js

```diff
@@ -226,6 +234,83 @@ function getBalance(req, res, next) {
     .done();
 }
 
+/**
```

Comment:
This logic should be in the fiscal year controller file.

Reply:
Will do! 👍